alacritty_terminal/tests/ref.rs

```diff
@@ -54,6 +54,12 @@ ref_tests! {
     zerowidth
     selective_erasure
     colored_reset
+    delete_chars_reset
+    alt_reset
+    deccolm_reset
+    decaln_reset
```

Comment:
On `decaln` I've followed `vte` behavior, since `xterm` one was a bit strange here. Other then that I've been testing `vte` and `xterm`, sometimes `urxvt`.

Reply:
Could you go into more detail what difference you encountered and why VTE makes more sense?

I ran `decaln` myself using `echo -e "\e#8"` and wasn't able to see any difference.

Reply:
Try the following in xterm. It should remove underline/strikeout effect.
```
printf "\e[31;1;4;9mTEST asd\n"
echo -e "\033#8"
```